sources/europe/hr/dgu-dof-2019-s-e.geojson

```diff
@@ -0,0 +1,656 @@
+{
+    "type": "Feature",
+    "properties": {
+        "category": "photo",
+        "id": "dgu-dof-2019-s-e",
+        "name": "dgu.hr: Croatia 2019 South+East Aerial imagery",
+        "icon": "https://osmlab.github.io/editor-layer-index/sources/europe/hr/dgu.png",
+        "start_date": "2019",
+        "end_date": "2019",
+        "url": "https://geoportal.dgu.hr/services/inspire/orthophoto_2019/ows?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap&LAYERS=OI.OrthoimageCoverage&STYLES=&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
+        "permission_osm": "explicit",
```

Comment:
I think explicit is where you have some documentation from the copyright holder that they are explicitly allowing use in OpenStreetMap. If you're just relying on a law that says there are no copyright protections etc. that wouldn't be explicit permission, it would be implicit.

Reply:
Yes. Example for explizit: https://wiki.openstreetmap.org/w/images/0/04/OSM_waiver_AirborneResearchSouthAustralia.pdf

Reply:
This happened before I got involved with the project, but here is my understanding. A new law (referenced above) was passed which regulated usage of public GIS data provided by the government. This law allowed us to use the data from DGU (government's geodesic administration), but the DGU's web page terms of service were in opposition with the law - denying any such use. We requested them to change the terms referencing the new law and they responded positively. I believe this response is what's considered to be explicit. However, I only found a blog post talking about this, and not the actual document.

I'll ask @hbogner to chime in here, since he was most involved in this.

Reply:
DGU is a copyright holder which sells the data, and they show it on https://geoportal.dgu.hr/ which is online for years now
Only problem is that there were strict copyrights on geoportal data, which prohibited use, copy, derivateive, ... anything usefull.
Years ago I requested(based on the freedom of information act) access to that data and was turned down every time.
Laste year they published an article about data being available for anonymous users: https://dgu.gov.hr/vijesti/mrezne-usluge-prostornih-podataka-drzavne-geodetske-uprave/5015
After they published that article I tried again and this time they accepted my request and changed the rights to use that data available for anonymous use.
Here is my explicit request to use that data: https://dgu.gov.hr/vijesti/mrezne-usluge-prostornih-podataka-drzavne-geodetske-uprave/5015

I fought with DGU for years on the rights to use the data tax payers already payed, and now I have to fight here again about the wording...

There are all the texts and you can read them and advise us is it explicit or implicit or apache attack helicopter or whatever, and we'll change it on all the sources, just let us use them in peace...

I think I already had similar discussion with @grischard @simonpoole about the DGU data, i may be mistaken about people included, and can't find the discussions now...

Reply:
Thanks for the replies with an explanation.

In my jurisdiction I can ask for data through a freedom of information request and receive the data but that doesn't mean it's free of copyright and I can do what I want with it.

> Temeljem Vaše zahtjeva od 12.3.2019. godine, te nadopune zahtjeva od 19.3.2019. godine, sukladno čl.27. st.3., odnosno čl. 23.st.1. t. 3. Zakona o pravu na pristup informacijama („Narodne Novine“, broj 25/13. i 85/15.) obavještavamo Vas da su na Geoportalu Državne geodetske uprave na poveznici [1] https://geoportal.dgu.hr/#/menu/uvjeti-koristenja objavljeni novi uvjeti korištenja koji omogućavaju ponovnu uporabu informacija objavljenih putem mrežnih usluga prostornih podataka Državne geodetske uprave.

The link there says "All published data is for informational purposes only and is for personal use and may not be used for commercial purposes or distributed to a third party" which is not compatible. What have I missed here? Is it still the case that they are claiming copyright or other restrictive terms of use in conflict with the law you mentioned that says they must have it open?

When I go through to https://geoportal.dgu.hr/#/menu/uvjeti-koristenja it says (translated)

_The online review service is available to anonymous and registered users free of charge. The user can use this service to create new value-added products. By accessing the online viewing service and the data it serves, the user acquires the right of use, without in any way transferring the right of ownership over the same. When publishing data, the user is obliged to point out the source of data as follows: "It contains data from the State Geodetic Administration, type of data, date of download and date to which the state of the data refers"._

So this appears to me like you can use for whatever you like but you need to provide attribution. Mostly where attribution is requested we like to confirm that attribution on the Contributors wiki page is sufficient, since we don't ask downstream OSM users to attribute all OSM sources.

@simonpoole does this sound right, or did you have any prior discussions on this?

> I fought with DGU for years on the rights to use the data tax payers already payed, and now I have to fight here again about the wording...

I'm just trying to to the due diligence to make sure that all the necessary legal permission is well documented so if there's ever any questions in the future about using this data we can point to it, if it turns out that an imagery layer we can't use makes it into the editors then the effort to go through and redact it all back out of OSM would be enormous, so just trying to avoid that.

> I think I already had similar discussion with @grischard @simonpoole about the DGU data, i may be mistaken about people included, and can't find the discussions now...

Sorry you can't find the prior discussions as those would be most helpful and avoid needing to go through it all again.

Reply:
Hi @andrewharvey (I think your second URL was supposed to be https://geoportal.dgu.hr/#/menu/uvjeti-koristenja-mreznih-usluga and not the same as the first one). If I understand correctly, only thing you find lacking is acknowledgement that DGU is OK with being attributed at https://wiki.openstreetmap.org/wiki/Contributors#Croatia (instead of somewhere else)? 

If that is it, I may point to look at the official correspondence of Hrvoje Bogner with DGU logged via public service [here](http://imamopravoznati.org/request/pravo_na_anonimno_koristenje_mre#outgoing-9457), where the explicit attribution location was stated, and DGU did not raise any complaints to it, on 16. April 2019.:

> Zahvaljujem na pozitivnom odgovoru.
> Kao što je specificirano u novo kreiranim uvjetima korištenja mrežnih usluga navest će se DGU kao izvor podataka na stranici projekta:
> https://wiki.openstreetmap.org/wiki/Contributors#Croatia

which translates as:

> I thank you for your positive answer.
> As specified in newly created terms of using network services, DGU will be attributed as data source at project page:
> https://wiki.openstreetmap.org/wiki/Contributors#Croatia

does that answer the concerns raised?

Reply:
Okay I'll take your word on it.

Reply:
They were informed about out intent to use it for OSM and as Matija said, they were sent a link to contributor pages.

Sorry for my frustration, but you were asking about specific terms of use and licenses, which is funny for Croatia.
Most institutions in Croatia which publish the data don't understand licenses.
Some publish the data online under strict copyright with no reuse, no derivative works, ... and when you ask them for the right to use the data they say: "It's online just download it..." 
Some are old school ex communist mentality where everything is classified and top secret, especially geospatial data, and you have to explain to them that things have changed and we are in EU ...
I think you get the picture ... :)

Thx for the merge.

Reply:
For the record and future reference: Hrvoje's understanding of EU law is correct, and I agree with his practical interpretation of it.

Reply:
Thanks for the comments, this does help, and if this is documented on the wiki, we can just link to it, now and for all future layers added under the same licensing conditions.